pkg/cephfs/controllerserver.go

```diff
@@ -44,6 +44,34 @@ var (
 	mtxControllerVolumeID = keymutex.NewHashed(0)
 )
 
+// ExpandVolume expand CephFS Volumes on demand based on resizer request
+func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
+
+	if err := cs.validateExpandVolumeRequest(req); err != nil {
+		klog.Errorf("ControllerExpandVolumeRequest validation failed: %v", err)
+		return nil, err
+	}
+
+	volID := req.GetVolumeId()
```

Comment:
Should we allow volume expansion for volumes that CSI did not provision? IOW, when a volume create came in with the provision as false?